src/messages/lang.json

```diff
@@ -1,3 +1,3 @@
 {
-    "langInstructions": "<p>Welcome!</p> <p>If you need help at any time, press the H key. To continue, select the start button below.</p>"
+    "langInstructions": "<p>Welcome!</p> <p>This tool will guide you through setting up your computer the way you like best.</p><p>For help at any time, select the help button or press the H key. To continue, select the next button or press the N key.</p>"
```

Comment:
Could you please add a space between tags e.g. `<p></p> <p></p>` this will help the self voicing. Same for the congratulations.json file.